span_test.go

```diff
@@ -3,6 +3,7 @@ package basictracer
 import (
 	"testing"
 
+	opentracing "github.com/opentracing/opentracing-go"
```

Comment:
FTR, the reason for it is that for a long time Godef would be unable to find a package if its path name was different, so yeah that's the reason for the import statement.